org.sonarlint.eclipse.site/pom.xml

```diff
@@ -27,4 +27,37 @@
     </plugins>
   </build>
 
+  <profiles>
+    <profile>
+      <id>coverage</id>
+      <dependencies>
+        <dependency>
```

Comment:
I would move this profile to the tests module, in order to avoid having to add this dependency.